### PR TITLE
Add dependencies to run sysvinit tasks on systemd systems

### PR DESCRIPTION
**Pull request by renanrodrigo**
Bug: 1763337

### CODE REVIEW

**Comment by StykMartin** [comment]
Missing Bug ID :eyes:

File: virt/image-install/Makefile [hunk 1]
```diff
@@ -119,5 +119,8 @@ $(METADATA):
 	@echo "Requires:	@virtualization-client" >> $(METADATA)
 	@echo "Requires:	@x11" >> $(METADATA)
 	@echo "Requires:	@fonts" >> $(METADATA)
+	@echo "Requires:	initscripts" >> $(METADATA)
+	@echo "Requires:	chkconfig" >> $(METADATA)
+	@echo "Requires:	iptables-servce" >> $(METADATA)
```
Comment (StykMartin) [inline virt/image-install/Makefile]: Do we even need this service? + typo

File: virt/install/Makefile [hunk 2]
```diff
@@ -124,5 +124,8 @@ $(METADATA):
 	@echo "Requires:	@x11" >> $(METADATA)
 	@echo "Requires:	@fonts" >> $(METADATA)
 	@echo "RunFor:		xen-libs"	>> $(METADATA)
+	@echo "Requires:	initscripts" >> $(METADATA)
+	@echo "Requires:	chkconfig" >> $(METADATA)
+	@echo "Requires:	iptables-servce" >> $(METADATA)
```
Comment (StykMartin) [inline virt/install/Makefile]: Do we even need this service? + typo

**Review by cbouchar — CHANGES_REQUESTED** [review]
should be iptables-service NOT iptables-servce in both locations

**Review by StykMartin — APPROVED** [review]
Looks good.
Did you try both **image-install** and **install** ?

**Comment by renanrodrigo** [comment]
I tried virt/install so far.
The iptables-service line wasn't necessary after all.
I'm running virt/image-install to check results.

**Review by cbouchar — APPROVED** [review]

**Pull request merged**